sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/models/CreateChatThreadOptions.java

```diff
@@ -92,7 +144,7 @@ public CreateChatThreadOptions setIdempotencyToken(String idempotencyToken) {
     }
 
     /**
-     * Creates a new instance of CreateChatThreadOptions
+     * Creates a new instance of CreateChatThreadOptions only providing topic parameter.
```

Comment:
nit; I don't think you need to specify the latter if there are no other constructors for this class.